src/QtAVPlayer/qavplayer.cpp

```diff
@@ -1313,6 +1313,19 @@ void QAVPlayer::setInputOptions(const QMap<QString, QString>  &opts)
     Q_EMIT inputOptionsChanged(opts);
 }
 
+
+/*!
+ * \brief Use to set log level of FFmpeg backend
+ * \param[in] level
+ * Level log to use. Please see:
+ * https://ffmpeg.org/doxygen/trunk/group__lavu__log__constants.html
+ * for value details
+ */
+void QAVPlayer::setLogsLevelBackend(int level)
```

Comment:
forgotten header?

Reply:
Oups, sorry :/ The header file was not stashed, I have fix it